### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by AdrianRaFo**
Update few versions including the upgrade to Hood 0.4.0 moving all benchmarks to json

### CODE REVIEW

File: helios-benchmarks/build.gradle [hunk 1]
```diff
@@ -39,43 +39,44 @@ jmh {
...
 compareBenchmark {
     previousBenchmarkPath = file("$rootDir/benchmarks/master_benchmark.csv")
-    currentBenchmarkPath = [file("build/reports/helios_benchmark.csv")]
-    threshold = 600
+    currentBenchmarkPath = [file("build/reports/helios_benchmark.json")]
+    outputFormat = "json"
 }
 
 compareBenchmarkCI {
     previousBenchmarkPath = file("$rootDir/benchmarks/master_benchmark.csv")
```
Comment (AdrianRaFo) [inline helios-benchmarks/build.gradle]: Realize the cross-comparison feature worked perfectly

**Comment by 47degdev** [comment]
***Hood benchmark comparison:***
:red_circle: `Decoding` (Threshold: 108.423407)

Benchmark | Value
|--|--|
master_benchmark | 11585.823369
helios_benchmark | 11456.104502757325


:red_circle: `Parsing` (Threshold: 760.098092)

Benchmark | Value
|--|--|
master_benchmark | 23084.587761
helios_benchmark | 22162.862566601543

**Comment by nomisRev** [comment]
This is awesome!

**Comment by AdrianRaFo** [comment]
Looks like json benchmarks are a little slower :joy:

**Comment by fedefernandez** [comment]
@AdrianRaFo can we try to generate the master benchmarks as part of the travis build?

**Comment by AdrianRaFo** [comment]
Set as part of this https://github.com/47deg/helios/issues/49

**Pull request merged**